### PR TITLE
Feat/overhauled server

### LINKED ISSUE #76: Scrape events from 3 pages of Eventbrite events webpage
**Feature purpose**
Scrape 60 or more events in total for each location

**Ideal solution**
Use puppeteer to move onto next events page and scrape events from there after scraping all 20 events from the original page.

**Alternative solution**
Use puppeteer-cluster to make multiple api calls (one for each page)

**Additional context**
We need to scrape 60 or more events for each location in order to populate the events map on ActNow. Right now, the scraper only scrapes the entirety of one events page and we need to figure out how to scrape the other pages containing more events.

### PR DESCRIPTION
**Pull request by lx1922**
### Issue
Closes #76

### What is changed / added
- Changed scraper to now use puppeteer cluster
- Cluster instances scrape different pages to scrape more than 60 events concurrently

### Screenshots
- https://user-images.githubusercontent.com/43979159/140854714-7053f33a-71df-45b9-923c-8a73fec34351.mp4

### CODE REVIEW

File: src/platform/functions/index.js [hunk 1]
```diff
@@ -1,103 +1,128 @@
...
 admin.initializeApp();
 
 const timeout = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
 
 exports.scrapeEventbrite = functions
   .runWith({
-    timeoutSeconds: 30,
-    memory: '1GB',
+    timeoutSeconds: 120,
+    memory: '2GB',
```
Comment (TheFatPanda97) [inline src/platform/functions/index.js:13]: let's not increase this
```suggestion
    timeoutSeconds: 60,
    memory: '1GB',
```

Let's only increase the timeoutseconds. I think 1 gb of ram should be enough

**Review by TheFatPanda97 — CHANGES_REQUESTED** [review]
almost good, just small for function memory usage

**Comment by lx1922** [comment]
Oh ok 👍 can you just approve this, I'll change memory in the next pr.

**Comment by TheFatPanda97** [comment]
> Oh ok 👍 can you just approve this, I'll change memory in the next pr.

for better tracking let's just make the change on this PR, you could just commit the suggestion I made

**Comment by lx1922** [comment]
> > Oh ok 👍 can you just approve this, I'll change memory in the next pr.
> 
> for better tracking let's just make the change on this PR, you could just commit the suggestion I made

Done!

**Review by TheFatPanda97 — APPROVED** [review]

**Pull request merged**